spec/chatops_deployer/project_spec.rb

```diff
@@ -5,6 +5,20 @@
   let(:project) { ChatopsDeployer::Project.new(repo, 'branch') }
   before { project.setup_directory }
 
+  describe "parse git url" do
+    context 'when github repo is passed' do
+      it 'does not throw any error if git url is valid' do
+        expect{ project }.not_to raise_error
+      end
+    end
+    context 'when any other git url is passed is passed' do
+      subject(:project) { ChatopsDeployer::Project.new('https://example.com/organization/repo.git', 'branch') }
+      it 'does not throw any error if git url is valid' do
+        expect{ project }.not_to raise_error
+      end
+    end
+  end
```

Comment:
We are only testing that errors are not raised. What I would test is ->

1. When sending github url, organization and repo are parsed properly
2. When sending gitlab url, organization and repo are parsed properly